src/lib/components/Button.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+  export let href: string = ""
+  export let onClick: () => void = () => {}
+  export let disabled: boolean = false
+</script>
+
+{#if href}
+  <a
+    {href}
+    class="ctaBtn bg-transparent hover:bg-blue-tertiary text-white py-2 px-4 rounded-full mx-6 sm:mr-6"
```

Comment:
for consistency with the tailwind usage and buttons, if you have the time to turn button class into something global with tailwind's `@apply`?


example:
```
.button {
    @apply bg-transparent hover:bg-blue-tertiary text-white border border-current py-2 px-4 rounded-full mx-6 sm:mr-6;

}

```